### DISCUSSION TITLE
Any recommendation on how to create mocks?

### DISCUSSION THREAD

**Discussion by donferi** [ecyrbe/zodios#167] [Q&A]
Was wondering if anyone has any ideas of how to do mocking with zodios. I'm aware I can use something like `msw` but it has its own downsides. It would be nice to have a way to provide some mocks for use with jest without having to mock the module.

Would be interesting to make it work with something like https://www.npmjs.com/package/@anatine/zod-mock too

Am I making sense?

**Comment by ecyrbe** [comment]
Zodios is compatible with moxios for mocking since it's based on axios.
